packages/terra-list/src/ListItem.scss

```diff
@@ -24,19 +26,32 @@
     top: 50%;
     transform: translate(0, -50%);
   }
-  .terra-ListItem-chevron > * {color: $terra-grey-30;}
 }
 
-// Update with UX styling
 .terra-ListItem--selected {
-  background-color: rgba(239, 68, 0, 0.53);
+  background-color: $terra-list-item-selected-background-color;
+}
+
+.terra-List-divided .terra-ListItem--selected {
+  &.terra-ListItem {
+    border-color: $terra-list-item-selected-divider-color !important;
```

Comment:
The `!important` rule applies the selected divider color over any other border color applied on the list item. This ensures the selection style is applied over the non-selected style.